specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2022-11-15/mongorbac.json

```diff
@@ -406,7 +406,7 @@
           "description": "A user-friendly name for the Role Definition. Must be unique for the database account."
         },
         "type": {
-          "type": "string",
+          "type": "int",
           "enum": [
             "BuiltInRole",
             "CustomRole"
```

Comment:
Does it need to remove "modelAsString" or add "values" attribute like this [example](https://github.com/Azure/azure-rest-api-specs/blob/969fd0c2634fbcc1975d7abe3749330a5145a97c/specification/mixedreality/resource-manager/Microsoft.MixedReality/stable/2021-01-01/common.json#L41), right?

Reply:
Yes, we need to have this to minimize the changes in our SDKs.
For some SDKs, this might not be a breaking for them

Reply:
fixed